src/main/scala/swiss/dasch/domain/ImageService.scala

```diff
@@ -72,6 +86,40 @@ final case class ImageServiceLive(sipiClient: SipiClient, assetInfos: AssetInfoS
         .whenZIO(Files.notExists(derivativePath))
         .as(JpxDerivativeFile.unsafeFrom(derivativePath))
   }
+
+  override def getDimensions(derivative: JpxDerivativeFile): Task[Dimensions] = {
+    val path = derivative.toPath
+    sipiClient
+      .queryImageFile(path)
+      .map(out => out.stdOut.split('\n'))
+      .flatMap { lines =>
+        val width: Option[Int Refined Positive]  = lines
+          .find(_.startsWith("nx"))
+          .flatMap(_.split('=').lastOption.map(_.trim))
+          .flatMap(_.toIntOption)
+          .flatMap { i =>
+            val w: Either[String, Int Refined Positive] = refineV(i)
+            w.toOption
+          }
+        val height: Option[Int Refined Positive] = lines
+          .find(_.startsWith("ny"))
+          .flatMap(_.split('=').lastOption.map(_.trim))
+          .flatMap(_.toIntOption)
+          .flatMap { i =>
+            val w: Either[String, Int Refined Positive] = refineV(i)
+            w.toOption
+          }
+        val dim                                  = for {
```

Comment:
this is a weird indentation

Reply:
There is a difference in the formatters.

Thanks for pointing this out. I have opened https://github.com/dasch-swiss/dsp-ingest/pull/111 in order to align dsp-ingest and dsp-api.